docker-compose/README.md

```diff
@@ -29,4 +29,14 @@ Next, edit `docker-compose.yml`, replacing `{YOUR-DOMAIN-HERE}` with the domain
 
 Running `docker-compose up --build` will now create a container from the `1password/scim` image. A redis container will also be started automatically to be used by the SCIM bridge. _After the DNS record has been propogated_, you can continue setting up your IdP with the SCIM bridge Administration Guide while monitoring the logs from the bridge on your local machine.
 
-Once you have tested the configuration, the bridge can be exited using ctrl/cmd-c, and restarted in daemon mode using `docker-compose up -d, or deployed for production use with Docker Swarm or Kubernetes. You can access logs using `docker-compose logs` at any point in the future.
\ No newline at end of file
+Once you have tested the configuration, the bridge can be exited using ctrl/cmd-c, and restarted in daemon mode using `docker-compose up -d, or deployed for production use with Docker Swarm or Kubernetes. You can access logs using `docker-compose logs` at any point in the future.
+
+## Automatically starting the SCIM bridge upon startup
+
+In order to automatically start the SCIM bridge upon startup when using docker-compose you'll need to automatically start the Docker daemon, then start op-scim.
+
+### Systemd
+
+* Enable dockerd to run on startup: `systemctl enable dockerd`
```

Comment:
i would re-order things in here, saying:
when systemd service (unit) file is configured reload systemd:
`systemctl daemon-reload`
and then enable the service with:
`systemctl enable op-scim`

Reply:
Not 100% sure I follow. do you mean tht the steps should be:

1. Install the service file (with edits)
2. reload systemd: `systemctl daemon-reload`
3. enable the op-scim service: `systemctl enable op-scim` 
?

Reply:
yes, because whenever we make changes to service file or adding a new one systemd needs to re-read changes

Reply:
thanks. will adjust.